addOns/zest/src/main/java/org/zaproxy/zap/extension/zest/ExtensionZest.java

```diff
@@ -583,10 +596,37 @@ public void updated(ScriptNode node) {
         if (this.getExtScript().getScriptUI() != null
                 && this.getExtScript().getScriptUI().isScriptDisplayed(sw)) {
             // We need to do this to prevent the UI slating any changes
+            displayScript(sw, allowFocus);
+        }
+    }
+
+    private void displayScript(ZestScriptWrapper sw, boolean allowFocus) {
+        try {
+            Method displayScriptMethod =
```

Comment:
Cache the methods and only keep calling them if not null.

Reply:
okay

Reply:
I was suggesting caching them earlier (e.g. constructor), as is it's still trying to get them always. This is to handle 2.13 which will not have the methods (so we shouldn't keep trying to get them each time).

Reply:
okay